### ISSUE TITLE
File type icons in Files pane, other dialogs, have no accessibility text

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#4988]
### System details

    RStudio Edition : Server
    RStudio Version : 1.3.264
    OS Version      : Chrome on MacOS hitting RStudio Server
    R Version       : any

### Steps to reproduce the problem
Using VoiceOver, get focus on the Files pane. Use VoiceOver keys to navigate to the column with file icons.

### Describe the problem in detail
The icons should describe what they are (folder, R Markdown, etc.)

### Describe the behavior you expected
The icons have no alt text so cause the image filenames to be read. These are long garbled filenames. For example, the folder icon in row three of my files pane is read as follows by VoiceOver:

"row 3 /Lt8WJRscHbhvPQAAAABJRU5ErkJggg== image"

### Additional Symptoms
When doing basic automated testing of RStudio startup view (including Files pane) each icon in the Files pane is flagged as an accessibility violation due to having no alt-text.
